misc_python/jobparser.py

```diff
@@ -267,7 +459,42 @@ def __repr__(self):
         lines += "Working Directory: {0}\n".format(self.WorkingDirectory)
         lines += "User: {0}\n".format(self.User)
         lines += "Comment: {0}\n".format(self.Comment)
-        lines += "Scheduled Date: {0}\n".format(self.ScheduledDate)
+#        lines += "User Data Size: {0}\n".format(self.UserDataSize)
```

Comment:
Might as well remove commented out code if not needed.

Reply:
Code removed.